docs/commands/ft.create.md

```diff
@@ -1,238 +1,270 @@
+  created with less than 32 text attributes.
+</details>
 
-* **SCHEMA {identifier} AS {attribute} {attribute type} {options...}**: After the SCHEMA keyword, we declare which fields to index:
+<details open>
+<summary><code>NOOFFSETS</code></summary> 
 
-    * **{identifier}**
+does not store term offsets for documents. It saves memory, but does not
+  allow exact searches or highlighting. It implies `NOHL`.
+</details>
 
-      For hashes, the identifier is a field name within the hash.
-      For JSON, the identifier is a JSON Path expression.
+<details open>
+<summary><code>TEMPORARY</code></summary> 
 
-    * **AS {attribute}**
+creates a lightweight temporary index that expires after a specified period of inactivity. The internal idle timer is reset whenever the index is searched or added to. Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore, you should consider using `SKIPINITIALSCAN` to avoid costly scanning.
 
-      This optional parameter defines the attribute associated to the identifier.
-      For example, you can use this feature to alias a complex JSONPath expression with more memorable (and easier to type) name
+When dropped, a temporary index does not delete the hashes as they may have been indexed in several indexes. Adding the `DD` flag deletes the hashes as well.
+</details>
 
-    #### Field Types
+<details open>
+<summary><code>NOHL</code></summary> 
 
-    * **TEXT**
+conserves storage space and memory by disabling highlighting support. If set, the corresponding byte offsets for term positions are not stored. `NOHL` is also implied by `NOOFFSETS`.
+</details>
 
-      Allows full-text search queries against the value in this attribute.
+<details open>
+<summary><code>NOFIELD</code></summary> 
 
-    * **TAG**
+does not store attribute bits for each term. It saves memory, but it does not allow
+  filtering by specific attributes.
+</details>
+
+<details open>
+<summary><code>NOFREQS</code></summary> 
 
-      Allows exact-match queries, such as categories or primary keys, against the value in this attribute. For more information, see [Tag Fields](/redisearch/reference/tags).
+avoids saving the term frequencies in the index. It saves
+  memory, but does not allow sorting based on the frequencies of a given term within the document.
+</details>
 
-    * **NUMERIC**
+<details open>
+<summary><code>STOPWORDS</code></summary> 
 
-      Allows numeric range queries against the value in this attribute. See [query syntax docs](/redisearch/reference/query_syntax) for details on how to use numeric ranges.
+sets the index with a custom stopword list, to be ignored during
+  indexing and search time. `{num}` is the number of stopwords, followed by a list of stopword arguments exactly the length of `{num}`.
```

Comment:
Same comment as this [one](https://github.com/RediSearch/RediSearch/pull/2959#discussion_r937743688)

Reply:
Thanks for bringing it here.